src/profile/helpers/editData.ts

```diff
@@ -150,17 +182,254 @@ function pickSources(
   }
 }
 
+export function cloneAndMutateItem(
+  data: EditData,
+  overrides?: Partial<EditData>
+): EditData {
+  return {
+    ...data,
+    ...overrides,
+  };
+}
+
+export function findItemIndex(
+  allItems: EditData[],
+  idOrEditData: string | EditData
+): number {
+  const itemId =
+    typeof idOrEditData === 'string' ? idOrEditData : idOrEditData.id;
+  return allItems.findIndex(item => itemId === item.id);
+}
+
+function findItem(allItems: EditData[], id: string): EditData | undefined {
+  return allItems.find(item => id === item.id);
+}
+
+export function updateItemAndCloneList(
```

Comment:
Did not find any usages outside of this file, so maybe the export is unnecessary?

Reply:
Removed.